packages/blockchain-link-utils/src/solana.ts

```diff
@@ -0,0 +1,205 @@
+import { ParsedInstruction, ParsedTransactionWithMeta } from '@solana/web3.js';
```

Comment:
for context this package here should contain anything that related to solana, that might be reusable at some point later. for example, we use to blockbook utils to transform blocbkook transaction format to suite transaction format. this also means that if there is some code here wich is very unlikely not reusable, it can maybe stay in blockchain-link package.

Reply:
I see, do you have some example of functionality which is here and should not. right now its basically just `transformTransaction` and stuff it directly needs. Also, I exported types from `blockchain-link-utils` so I could remove solana/web3.js dependency altogether